lib/matplotlib/patches.py

```diff
@@ -3542,10 +3560,25 @@ def __init__(self,
                 Orientation of the bracket, as a counterclockwise angle.
                 0 degrees means perpendicular to the line.
             """
-            super().__init__(True, True,
-                             widthA=widthA, lengthA=0, angleA=angleA,
+            super().__init__(widthA=widthA, lengthA=0, angleA=angleA,
                              widthB=widthB, lengthB=0, angleB=angleB)
 
+    @_register_style(_style_list, name=']->')
+    class BracketCurve(_Curve):
+        beginarrow_bracket = endarrow_curve = True
+
+        def __init__(self, widthA=1., lengthA=0.2, angleA=0):
```

Comment:
Why is this `angleA` default different from the `angleB` below?

Reply:
`0` and `None` will do the same thing for `angleA` and `angleB`. To maintain the pattern I changed the `angleA=0` to `angleA=None`